src/CyberdropDownloader.Core/FileDownloader.cs

```diff
@@ -86,13 +86,17 @@ public static long ConvertAlbumSizeToByte(string albumSize)
             else if (albumSize.Contains("TB"))
                 byteValue = 1099511627776;
 
-            return int.Parse(Regex.Replace(albumSize, "[^0-9]+", string.Empty)) * byteValue;
```

Comment:
Wasn't matching the decimal point which was causing the returning value to be WAY higher than expected.